includes/admin/class-amp-post-meta-box.php

```diff
@@ -163,6 +163,11 @@ public function enqueue_admin_assets() {
 
 		wp_styles()->add_data( self::ASSETS_HANDLE, 'rtl', 'replace' );
 
+		// Abort if version of WordPress is too old.
+		if ( ! Services::get( 'dependency_support' )->has_support_from_core() ) {
```

Comment:
I chose to go with checking for core support here rather than support in general because I figure it would be a bad idea to try to `amp_register_polyfills` on the edit post screen, not knowing what possible conflicts could result.